ionic.project

```diff
@@ -1,4 +1,4 @@
 {
   "name": "poc",
-  "app_id": "6b6d1090"
+  "app_id": "b1783953"
```

Comment:
this is interesting, as you've deployed to the Ionic View the `app_id` was overwritten, need to figure how to get around this